tests/subsys/mgmt/mcumgr/os_mgmt_info/testcase.yaml

```diff
@@ -17,17 +17,17 @@ common:
     - mips
     - posix
 tests:
-  os.mgmt.info: {}
-  os.mgmt.info_no_hooks:
+  mgmt.mcumgr.info: {}
```

Comment:
@nordicjm I do not think this is correct change, probably this should be "mgmt.mcumgr.os".